### ISSUE TITLE
Model not converging?

### ISSUE THREAD

**Issue by pablogranolabar** [guxd/DialogBERT#8]
Using the standard main.py training loop, I've been training on V100 tiny for almost a week but without it stopping? Is there additional hyperparameter tuning needed even to run the tiny training process?

```
python3 main.py --model_size=tiny --per_gpu_train_batch_size=24

avg_len = 12.61646884272997
bleu = 0.03122757749152926
meteor = 0.039703799201764936
nist = 0.12024726693793758
perplexity = 116.93566131591797
rouge-L = 0.05778559382996833
valid_loss = 4.761623978844736
```

Can you share what your final numbers were after training tiny and small?

**Comment by guxd** [comment]
The program will not stop until reaching the maximum iterations. You should manually track the learning curves and stop training when needed.  We reported the optimal result at the iteration which shows the minimum validation loss.
